torch/_dynamo/variables/misc.py

```diff
@@ -1,6 +1,7 @@
 import collections
 import functools
 import inspect
+import sys
```

Comment:
not needed?

Reply:
`NullVariable` at the very bottom of this file requires `sys`. Wasn't caught before in CI because that code is only reached in 3.11. Adding here now so I don't forget later.